.github/workflows/release.yml

```diff
@@ -26,6 +26,15 @@ jobs:
       - name: Install gox
```

Comment:
I think we need to enable docker cli experiimental features: [like this](https://github.com/caarlos0/domain_exporter/blob/master/.github/workflows/build.yml#L19)

Reply:
DOCKER_CLI_EXPERIMENTAL may not need to be set with latest docker version